QEfficient/exporter/export_utils.py

```diff
@@ -43,7 +43,7 @@ def export_onnx(
     # Inspect the model's forward method arguments
     pt_model_code = pt_model.forward.__code__
     pt_input_names = pt_model_code.co_varnames[1 : pt_model_code.co_argcount]
-
+    starcoder1_inputs = False
```

Comment:
nit: is this conditioning necessary? can avoid. and check for `past_` only

Reply:
Only `past_` will be applicable to other models too right. So other models will error out.

Reply:
What is special about starcoder inputs? I guess it should work fine without adding `starcoder_1_inputs` variable.

Reply:
[BS, CTX_len,d_head] is the layout. ctx_len is in axis 1 while for other models it is in axis 2.